spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/connection/MockClientConnectionFactoryBuilder.java

```diff
@@ -26,12 +26,12 @@
  *
  */
 
-public class MockFactory {
-	private MockFactory() {
+public class MockClientConnectionFactoryBuilder {
```

Comment:
I am still against these new classes.
Please, make the logic as `private static` in those test classes affected.
Thanks